go/ephemeral/lib.go

```diff
@@ -57,6 +57,11 @@ func (e *EKLib) NewMetaContext(ctx context.Context) libkb.MetaContext {
 
 func (e *EKLib) checkLoginAndPUK(ctx context.Context) (loggedIn bool, err error) {
 	m := e.NewMetaContext(ctx)
+	if oneshot, err := e.G().IsOneshot(ctx); err != nil || oneshot {
+		e.G().Log.CDebugf(ctx, "EKLib#ShouldRun failed: %s, isOneshot: %v", err, oneshot)
```

Comment:
`s/EKLib#ShouldRun/checkLoginAndPuk/`. Also this is going to say "failed" even when there was no error. Maybe tweak the wording?